starter-code/ShoppingListVer2/app/src/main/java/ly/generalassemb/drewmahrt/shoppinglistver2/GrocerySQLHelper.java

```diff
@@ -0,0 +1,81 @@
+package ly.generalassemb.drewmahrt.shoppinglistver2;
+
+import android.content.Context;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+import android.database.sqlite.SQLiteOpenHelper;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by NikitaShuvalov on 10/25/16.
+ */
+
+
+
+public class GrocerySQLHelper extends SQLiteOpenHelper {
+    public static final String DATABASE_NAME = "SHOPPING_DB";
+    public static final String TABLE_NAME = "SHOPPING_LIST";
+    public static final String COL_ID = "_id";
+    public static final String COL_NAME = "ITEM_NAME";
+    public static final String COL_DESCRIPTION = "DESCRIPTION";
+    public static final String COL_PRICE = "PRICE";
+    public static final String COL_TYPE = "TYPE";
+
+    private static final String CREATE_TABLE_EXE = "CREATE TABLE "+
+            TABLE_NAME + " ("+ COL_ID +" TEXT, "+ COL_NAME+ " TEXT, " + COL_DESCRIPTION + " TEXT, " + COL_PRICE + " TEXT, "+ COL_TYPE+ " TEXT)";
+
+    private static GrocerySQLHelper myInstance;
+
+    public static GrocerySQLHelper getInstance(Context context){
+        if (myInstance == null){
+            myInstance = new GrocerySQLHelper(context.getApplicationContext());
+        }
+        return myInstance;
+    }
+
+    private GrocerySQLHelper(Context context){
+        super(context, DATABASE_NAME, null,7);
+
+    }
+    @Override
+    public void onCreate(SQLiteDatabase sqLiteDatabase) { sqLiteDatabase.execSQL(CREATE_TABLE_EXE);
```

Comment:
It's more readable if you don't put the method body on the same line as the signature - always put a line break after the opening `{` of the method body